TrustnetAuthenticWebQ.md

```diff
@@ -189,8 +197,9 @@ way of identifying the block that needs to be annotated. There's no existing cro
 for this.
 
 Assessments are bound to URLs, but redirects, query parameters, and user-personalized URLs can all create vast numbers of 
-distinct URLs for the same content.   It would be beneficial to develop standards for indicating the "canonical" URL for a piece
-of content to help converge different assessments.   These standards should not simplify lying by those seeking to avoid assessments.
+distinct URLs for the same content.   There is a standard for indicating the "canonical" URL for a piece
+of content—[`<link rel=canonical>`](https://html.spec.whatwg.org/multipage/links.html#link-type-canonical)—whose integrity is reinforced by its effect on search rankings.
+However, as the Trustnet use may produce the opposite incentive for some content providers, work may be needed to maintain the signal.

```

Comment:
I'm not sure that this paragraph is needed at all given the existing standard for this. Also see https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls#use-rel=canonical-link-annotations